lib/matroska/MatroskaParser.ts

```diff
@@ -68,12 +67,18 @@ export class MatroskaParser extends BasicParser {
         const entries = audioTracks.entries.filter(entry => {
           return entry.trackType === TrackType.audio.valueOf();
         });
-        assert.equal(entries.length, 1, 'mapping limited to single audio track');
-        const audioTrack = entries[0];
 
-        this.metadata.setFormat('codec', audioTrack.codecID.replace('A_', ''));
-        this.metadata.setFormat('sampleRate', audioTrack.audio.samplingFrequency);
-        this.metadata.setFormat('numberOfChannels', audioTrack.audio.channels);
+        let audioTrack = entries.find(entry => entry.flagDefault);
```

Comment:
With reduce it can maybe be done slightly smarter, because it allows you to choose the best option between 2 audio tracks. A kind of sorting, based on criteria you define, and you pick the best match.